src/logger.js

```diff
@@ -176,14 +123,38 @@ class Logger {
     }
 }
 
+function getLogger() {
+    const { metadata, config } = getContainer();
+    const defaultLogglyOptions = {
+        enabled: false,
+        environment: 'dev',
+        subdomain: 'loggly',
+        token: 'token',
+    };
+
+    if (!config && !metadata) {
+        return createLogger(
+            'testing-logger',
+            'info',
+            defaultLogglyOptions,
+        );
+    }
+
+    if (metadata.testing) {
```

Comment:
This in particular means unit tests will never accidentally log to loggly, that the only real difference between the testing config and the normal config.